src/components/Doc/styles.module.scss

```diff
@@ -60,6 +73,13 @@ main.DocWrapper pre {
 
 main.DocWrapper a {
   font-weight: 700;
+  text-decoration: underline;
+  text-underline-offset: 4px;
+  transition: 0.2s;
```

Comment:
Optional: could we use the transition properties that'll change in this declaration? Minor perf improvement, transition: transform, is faster than the implicit transition: all

Reply:
didn't know that, will fix